kikar_hamedina/facebook_feeds/models.py

```diff
@@ -441,6 +441,17 @@ def is_youtube_video(self):
         return False
 
     @property
+    def video_source_type(self):
```

Comment:
Are we using this? Is this here on purpose? if so, how does it relate to the pull request?